platform/packages/currency/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub type SymbolOwned = String;
 // Not extending Serialize + DeserializeOwbed since the serde derive implementations fail to
 // satisfy trait bounds with regards of the lifetimes
 // Foe example, https://stackoverflow.com/questions/70774093/generic-type-that-implements-deserializeowned
-pub trait Currency: Copy + Ord + Default + Debug + 'static {
+pub trait Currency: Debug + Copy + Ord + JsonSchema + Sized + 'static {
```

Comment:
[same like the issues above about Group]

Reply:
As mentioned in the above comment. Those are common bounds, thus can be moved up to here. If we don't want those, we'd have to convert to manual implementations.